README.md

```diff
@@ -31,6 +33,9 @@ custom:
         - src/api
         - src/cow/*.js
       command: echo "api folder or js file in cow folder was modified!"
+    - path:
+        - src/**/**
+      command: "echo received $WATCHER_EVENT_TYPE event for $WATCHER_EVENT_PATH"
```

Comment:
Please can you add an example of exactly what this looks like. I think it would help people identify if it meets their needs, in particular how paths are formatted as that's not clear yet. Appreciate you've already specified the WATCHER_EVENT_TYPE type above.

```suggestion
      # this prints for example "received change event for src/path/to/file.ts"
      command: "echo received $WATCHER_EVENT_TYPE event for $WATCHER_EVENT_PATH"
```

nb: I _think_ this is the behaviour, based on my recollection of chokidar. But please correct me here if that's not how paths are actually formatted.